src/index.test.js

```diff
@@ -0,0 +1,64 @@
+window.jstz = require('esn-frontend-common-libs/src/frontend/components/jstzdetect/jstz.js');
+window.jQuery = require('jquery/dist/jquery.js');
+window.$ = window.jQuery;
+window.$q = require('q/q.js');
```

Comment:
Can't we rely on webpack for this?

Reply:
I tried with webpack but some throw errors when using with `$window` of AngularJS. Therefore I have to put them here. Will try to refactor this once every test run smoothly.